CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Change history for ui-eholdings
 
+## [5.1.0] (IN PROGRESS)
```

Comment:
Should it be bug fix release 5.0.1?

Reply:
We need to release this as a Q2 Hotfix, so it should be 4.0.2 version, but I'll update this in release branch b4.0